dev-docs/bidders/ix.md

```diff
@@ -272,47 +263,75 @@ ortb2Imp: {
+2. (Recommended) To force the Index renderer as the default renderer for Index, set `backupOnly` to true (see below example). This configuration allows you to use your own renderer only when an adapter does not provide their own renderer. This also forces any adapter that has a renderer to use their renderer by default. If no renderer is provided, by default,  we will use our own renderer script.
+3. Configure the player according to the options in the [Bid request parameters](#bid-request-parameters) section below.
+4. Depending on your existing Prebid setup, complete one of the following:
+    * If you have an existing Prebid.js integration for banner, you can use the corresponding line items in your Google Ad Manager (GAM) account to retrieve outstream video demand. For more information, see Prebid's documentation on [How to set up line items](https://docs.prebid.org/adops/step-by-step.html).
+    * If you do not have an existing Prebid.js integration for banner, create a line item in your GAM account that is capable of serving an outstream video ad. For more information about how to do this, see Prebid's documentation on Setting up [Prebid Video in Google Ad Manager](https://docs.prebid.org/adops/setting-up-prebid-video-in-dfp.html).
+5. Notify your Index Representative and provide your test page URL for validation. They will work with you to test the video player and confirm that outstream video ads are being retrieved.<br />
+
 For more information on how to structure the video object, refer to the following code example:<br />
 
 ```javascript
-var adUnits = [{
-    code: 'div-gpt-ad-1571167646410-1',
+pbjs.addAdUnit({
+    code: 'video1',
+    // This renderer would apply to all prebid creatives...
+    renderer: {
+        url: 'example.com/publishersCustomRenderer.js',
+	   backupOnly: true,
+        render: function(bid) { renderAdUnit(...)  }
+    },
     mediaTypes: {
+    // Pub renderer video settings...
         video: {
-            playerSize: [640, 360],
             context: 'outstream',
-            api: [2],
-            protocols: [2, 3, 5, 6],
-            minduration: 5,
-            maxduration: 30,
-            mimes: ['video/mp4', 'application/javascript'],
-            placement: 5
+            playerSize: [640, 480],
+            mimes: ['video/mp4'],
+            protocols: [1, 2, 3, 4, 5, 6, 7, 8],
+            playbackmethod: [2],
+            skip: 1
+            plcmt: 2
+        
         }
     },
     bids: [{
         bidder: 'ix',
         params: {
-            siteId: '715964'
+            siteId: '12345',
+		 // Index renderer video settings...
             video: {
-                playerConfig: {
-                    floatOnScroll: true,
-                    floatSize: [300,250]
-                }
+			 //minimum size for Video Player size is 144x144 in pixel
+			//maxduration can be any.
+		 context: 'outstream',
```

Comment:
Please update the alignment in this example

Reply:
done